### PR TITLE
docs(i18n): add Persian translation and support rtl `languageDirection`

### PR DESCRIPTION
**Pull request by Goudarz**
Complete Persian translation of the documentation and also adding the `languageDirection` method to the hugo.yaml file.

### CODE REVIEW

File: layouts/_default/baseof.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="{{ .Site.Language.Lang }}">
   {{- partial "head.html" . -}}
-  <body dir="ltr">
+  <body dir="{{ .Site.Language.LanguageDirection }}">
```
Comment (imfing) [inline layouts/_default/baseof.html:4]:
```suggestion
  <body dir="{{ .Site.Language.LanguageDirection | default `ltr` }}">
```

I think we need to default to `ltr` for backward compatibility

File: exampleSite/hugo.yaml [hunk 2]
```diff
@@ -20,13 +20,21 @@ defaultContentLanguage: en
 languages:
   en:
     languageName: English
+    languageDirection: ltr
```
Comment (imfing) [inline exampleSite/hugo.yaml:23]:
```suggestion
```

can be removed as we default to `ltr`

File: exampleSite/hugo.yaml [hunk 3]
```diff
@@ -20,13 +20,21 @@ defaultContentLanguage: en
...
     weight: 1
     title: Hextra
   zh-cn:
     languageName: 简体中文
     languageCode: zh-CN
+    languageDirection: ltr
```
Comment (imfing) [inline exampleSite/hugo.yaml:29]:
```suggestion
```

^ same

**Review by imfing — CHANGES_REQUESTED** [review]

File: layouts/_default/baseof.html (hunk 1 above)
Comment (Goudarz) [reply layouts/_default/baseof.html]: Ok, Updated the PR.

**Comment by imfing** [comment]
@Goudarz awesome job on the translations! Thanks for translating all the docs.

Didn't double-check the translations, but everything looks good on this link: [deploy-preview-301--hugo-hextra.netlify.app](https://deploy-preview-301--hugo-hextra.netlify.app/)

If you spot any issues with how the theme supports right-to-left languages, please drop an issue or PR, thanks!

**Comment by dogweather** [comment]
@Goudarz What do you think about this issue: do codeblocks on Persian sites have the same issue as Hebrew? https://github.com/imfing/hextra/discussions/347

**Pull request merged**